package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "metasync",
-  "version": "0.1.6",
+  "version": "0.1.7",
```

Comment:
Is there a reason in bumping the version? This commit introduces neither new functionality nor bugfixes. There's no point in releasing it so far, IMO.